src/services/cache-config/CacheConfigService.ts

```diff
@@ -375,7 +377,7 @@ export class CacheConfigService {
   private createCacheLoader(builder: CompositeOpBuilder, address: string[], config: any): void {
     let loaderClass: string = deepGet(config, "loader.LOADER.class");
     if (isNotNullOrUndefined(loaderClass) && loaderClass !== "None") {
-      this.createHelper(builder, address.concat("loader", "LOADER"), config.loader);
+      this.createHelper(builder, address.concat("loader", "LOADER"), config.loader, true);
```

Comment:
`createHelper` only has three params, not sure why you're passing a boolean here